ext/standard/streamsfuncs.c

```diff
@@ -129,26 +129,25 @@ PHP_FUNCTION(stream_socket_client)
 
 	context = php_stream_context_from_zval(zcontext, flags & PHP_FILE_NO_DEFAULT_CONTEXT);
 
-	if (flags & PHP_STREAM_CLIENT_PERSISTENT) {
-		spprintf(&hashkey, 0, "stream_socket_client__%s", ZSTR_VAL(host));
-	}
-
 	/* prepare the timeout value for use */
 	struct timeval *tv_pointer;
-	if (timeout < 0.0 || timeout >= (double) PHP_TIMEOUT_ULL_MAX / 1000000.0) {
```

Comment:
Do we no longer need the overflow check?

Reply:
We probably do, actually.